udaykalra/walkthroughs/week-3-server/examples/page-view-counter/target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
+#Mon Jun 29 23:09:39 UTC 2020
+version=1
+groupId=com.google.sps
+artifactId=page-view-counter
```

Comment:
There's a bunch more generated files (this and a number of files below). I updated the gitignore rule:
https://github.com/googleinterns/step174-2020/pull/26

Here's how to make git forget these files:
https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore

Reply:
This is an example demo, not the webpage.